functions_lab.js

```diff
@@ -0,0 +1,418 @@
+  }
+}
+console.log(largestOfTwoNum(num1, num2))
+
+// #########
+
+const largestOfTwoNum = (num1, num2) => {
+  if (num1 < num2) {
+    return num2
+  } else if (num1 > num2) {
+    return num1
+  } else {
+    return num1, num2
+  }
+}
+let num1 = 12;
+let num2 = 4
+console.log(largestOfTwoNum(num1, num2))
+
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's
+// age in human years. The function returns the dog's age in dog years.
+//The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+
+console.log('=======answer#2=========')
+
+function getDogAge (num) {
+return num * 7
+}
+let num = 4
+let result = num * 7
+console.log(result)
+
+//#################
+
+const getDogAge = (num) => {
+  return num * 7
+}
+let num = 4
+let result = num * 7
+console.log(result)
+//
+
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the
+// sum of all the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+
+console.log('=========answer#3=======')
+
+
+function arraySum (arr) {
+
+  return
+
+  i++
+}
+
+let arr = [13,2,4,-2];
+
+
+const arraySum = (arr) => {
+  let sum = 0;
+for (let i = 0; i < arr.length; i++ ) {
+  sum += arr[i]
+  }
+
+  return sum
+}
+
+
+console.log(arraySum(arr))
+
+//
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+
+console.log('===========answer#4========')
+
+function sumOfTwoNums (num1, num2) {
+  return num1 + num2
+}
+let result1  = sumOfTwoNums(100, 2)
+console.log(result1)
+let result2 = sumOfTwoNums(14, 5)
+console.log(result2)
+
+
+//
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding
+//letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+//
+// console.log('=======answer#5=========')
+let num = 88
+const letterGrade = (num) => {
+  if (num >= 100) {
+    return "A+"
+  } else if (num <=99  && num >= 90){
+    return "A"
+  } else if (num >= 80 && num <= 89){
+    return "B"
+  } else if (num >= 70 && num <= 79) {
+    return "C"
+  } else if (num >= 65 && num <= 69){
+    return "D"
+  } else {
+    return "F"
+}
+}
+// let num = 88
+let result = letterGrade(num)
+
+
+console.log(result)
+
+
+// ## Question 6
+//
+// Implement a function named `repeatLog` that takes a string and a number
+//as parameters. The function should log `message` `count` number of times.
+//
+// Example:
+//
+// Input: `"+", 10`
+//
+// Output: `++++++++++`
+
+console.log('==========answer#6=======')
+// //
+function repeatLog (param1) {
+  return x = x.repeat(param1)
+  }
+let x = "+"
+let result = x.repeat(10)
```

Comment:
Again just use the function that you wrote. No need for let result = x.repeat(10)